### ISSUE TITLE
More Issues with 5.2.5

### ISSUE THREAD

**Issue by sh1mrr4** [GC-spigot/AdvancedEnchantments#345]
Hey,

we just finished testing some of the changes of the new update and it seems that some issues still exist:

- TRENCH: In combination with fortune, the fortune effect doesn't apply to the initial block broken when mining ores (e.g. for coal ore)

- TRENCH: When breaking iron/gold ore, the ores don't drop at all (aside from the initial broken block)

- SMELT: Not sure if this is intended, but when you mine iron/gold ore with fortune and autosmelt, the ores always drop the same number of ingots, corresponding to the fortune level (e.g. when using Fortune 3, ores will always drop three ingots), instead of being randomized like mining coal ore would be.

- Also, since the last update, every time we enchant an item with a book, it creates a ghost-copy of the item in your inventory without the new enchantment on it. The extra item disappears after relogging or throwing it away, so it isn't game-breaking, but it is annoying nonetheless.

AdvancedEnchantments version: v5.2.5
PaperSpigot: 1.13.2

Thank you!

**Comment by Daphsquid** [comment]
Hi, I work on the same server as @sh1mrr4 (he is asleep right now). 

> * Also, since the last update, every time we enchant an item with a book, it creates a ghost-copy of the item in your inventory without the new enchantment on it. The extra item disappears after relogging or throwing it away, so it isn't game-breaking, but it is annoying nonetheless.

A few hours after reporting this issue, I was poking around on our server, and noticed that this "ghost-item" issue wasn't unique to AdvancedEnchantments at all. In fact, it was happening for every single plugin with a GUI (and just happened to be occurring when adding custom enchants on items). We've discovered that the issue is due to ProtocolSupport. We've removed this plugin and the issue is no longer occurring. 

Sorry about the confusion!
